products/sce/container-groups/registries/gcp-gar.mdx

```diff
@@ -11,23 +11,23 @@ the SaladCloud portal to interact with GCP through APIs, a JSON key for a servic
 2. Access the GCP [Console home page](https://console.cloud.google.com), Click on 'Go to project settings' to proceed.
 3. On the left navigation bar, find and select the 'Service Accounts' menu.
 4. Choose 'Create Service Account' to initiate the process.
-5. Provide a name and description for the service account, grant it the necessary access to your project, and proceed to
-   create the account.
+5. Provide a name and description for the service account, grant it `Artifact Registry Reader` access to your project, and proceed to
```

Comment:
Isn't `Artifact Registry Writer` also required, somehow? I thought that was an issue for someone.

Reply:
I tested with just Artifact Registry Read and it worked fine, my setup is very simple though, so maybe a more configurated setup/console locks stuff down somehow? Just double checked and Artifact Registry Read works fine...

For my testing:

Created a registry
pushed their example image
created service account with various roles
tested each role individually
only found that the basic - browser failed